htdocs/fourn/commande/dispatch.php

```diff
@@ -1128,16 +1128,15 @@
 	if ($conf->reception->enabled) {
 		$sql .= " ,cfd.fk_reception, r.date_delivery";
 	}
-	$sql .= " FROM ".MAIN_DB_PREFIX."product as p,";
-	$sql .= " ".MAIN_DB_PREFIX."commande_fournisseur_dispatch as cfd";
+	$sql .= " FROM ".MAIN_DB_PREFIX."commande_fournisseur_dispatch as cfd";
+	$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."product as p ON cfd.fk_product = p.rowid";
```

Comment:
Why adding a LEFT JOIN here and not INNER JOIN ?  
cfd.fk_product should always be defined to a value existing in product table.

Reply:
LEFT to change into INNER